pkg/repo/mysql/services.go

```diff
@@ -0,0 +1,90 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+)
+
+func (r *Repo) SaveService(ctx context.Context, service *model.Service) error {
+	query := `INSERT INTO service (id, name) VALUES (UUID_TO_BIN(?), ?) ON DUPLICATE KEY UPDATE name = VALUES (name)`
+
+	if service.ID == uuid.Nil {
+		service.ID = uuid.New()
+	}
+
+	_, err := r.db.ExecContext(ctx, query, service.ID, service.Name)
+	if err != nil {
+		return fmt.Errorf("db: insert service: %w", err)
```

Comment:
use `repo` everywhere in the error messages instead of `db`

```suggestion
		return fmt.Errorf("repo: insert service: %w", err)
```